iis-wf/iis-wf-metadataextraction/src/main/java/eu/dnetlib/iis/wf/metadataextraction/MetadataExtractorMapper.java

```diff
@@ -117,6 +146,16 @@ public class MetadataExtractorMapper extends Mapper<AvroKey<DocumentContent>, Nu
      */
     private ContentApprover contentApprover;
     
+    /**
+     * Grobid client responsible for communication with external grobid server.
+     */
+    private GrobidClient grobidClient;
+
+    /**
+     * Grobid server version.
+     */
+    private String grobidServerVersion;
```

Comment:
This information is also available in the Grobid results. should we try to ensure that they match? It seems that we do not have it in Grobid errors unfortunately so setting it in configuration it cannot probably be completely avoided.